dashboard/src/components/LinkWithIcon/LinkWithIcon.tsx

```diff
@@ -25,16 +27,18 @@ const LinkWithIcon = ({
       <span className="font-bold">
         <FormattedMessage id={title} />
       </span>
-      <WrapperLink
-        className="flex flex-row items-center gap-1"
-        href={link}
-        target="_blank"
-        rel="noreferrer"
-        onClick={onClick}
-      >
-        {linkText}
-        {icon}
-      </WrapperLink>
+      {linkComponent ?? (
```

Comment:
Why did you use this props?

Reply:
I'm using this prop so that I'm able to pass a custom Link component (in this case a tanstack router `Link` component) to the LinkWithIcon component, since it only render "simple" links (as an 'a' HTML Element)